apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -391,6 +395,86 @@ schema("/clients/:clientid/keepalive") ->
             }
         }
     };
+schema("/clients/:clientid/mqueue_messages") ->
```

Comment:
worth merging the common parts of the two clauses ?

Reply:
Done